### ISSUE TITLE
滚动NestedScrollView 的 body 下的Column 会报高度溢出 A RenderFlex overflowed by 40 pixels on the bottom.

### ISSUE THREAD

**Issue by purestart** [fluttercandies/extended_nested_scroll_view#67]
您好！

headerSliverBuilder 高度 超过 一屏的时候

extened_nested_scroll_view_demo 这个示例中 滚动NestedScrollView 的 body 下的Column 会报高度溢出 A RenderFlex overflowed by 40 pixels on the bottom.

这个问题麻烦看看，谢谢

**Comment by zmtzawqlp** [comment]
无所谓。不影响

**Comment by Dove56** [comment]
BoxConstraints has non-normalized height constraints.

**Comment by OHeroJ** [comment]
+1

**Comment by lvpeng0412** [comment]
tabbar包一个Flexible就不会报错了 @purestart

**Comment by techouse** [comment]
I've recently had to deal with a sliver header that was larger than the screen was displayed on.

I tried wrapping the `TabBar` into a `Flexible` which does resolve the overflow but it also makes the `TabBar` animate somewhat weirdly when it gets scrolled in.

The only way to resolve the overflow issue was to add the tab bar to the [`bottom`](https://api.flutter.dev/flutter/material/SliverAppBar/bottom.html) of my `SliverAppBar` effectively making it part of the header sliver and then Flutter stops complaining.
